objects/src/assets/token_symbol.rs

```diff
@@ -0,0 +1,79 @@
+use super::{AssetError, Felt, StarkField, ToString};
+use crate::utils::string::String;
+
+const MAX_SUMBOL_LENGHT: usize = 6;
+#[derive(Clone, Copy, Debug)]
+pub struct TokenSymbol(Felt);
```

Comment:
nit: I would probably move the constant into `TokenSymbol` implementation. For example:
```Rust
impl TokenSymbol {
    pub const MAX_SUMBOL_LENGHT: usize = 6;

    ... 
}
```

Reply:
I can't use the constant in `encode_symbol_to_felt()` if it is moved to `TokenSymbol`

Reply:
You should be able to use it as `TokenSymbol:: MAX_SUMBOL_LENGHT` - or does this not work for some reason?